apps/qubit/modules/search/templates/_updatesSearch.php

```diff
@@ -59,6 +59,23 @@
 
         </div>
 
+        <?php if (sfConfig::get('app_audit_log_enabled', false)): ?>
+          <div class="filter-row io-options">
+            <div class="filter-left">
+              <?php echo $form->user
+                ->label(__('User'))
+                ->renderLabel() ?>
+              <?php echo $form->user->render(array('class' => 'form-autocomplete')) ?>
+              <input class="list" type="hidden" value="<?php echo url_for(array('module' => 'user', 'action' => 'autocomplete')) ?>"/>
+
+              <?php if (isset($user)): ?>
+                <p>
+                  <em><?php echo __('Currently displaying:') ?> <?php echo $user->getUsername() ?></em>
```

Comment:
Nitpick: Should this formatting 'em' tag be done with css so that it can be more easily overridden in a custom theme?